jekyll/_cci2/server-3-operator-extending-internal-volumes.adoc

```diff
@@ -0,0 +1,157 @@
+---
+version:
+- Server v3.x
+- Server Admin
+---
+= CircleCI Server v3.x Internal Database Volume Expansion
+:page-layout: classic-docs
+:imagesdir: ../assets/img/docs/
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+_Persistent Volume Expansion of MongoDB and Postgres is available for server v3.2.0 and up._
+
+If you have choosen to deploy either of the CircleCI databases (MongoDB or Postgres) within the cluster, as opposed externally provisoning these databases, then there may come a point at which the storage space initially made available to these databases is no longer sufficient. Internal databases in your Kubernetes cluster make use of https://kubernetes.io/docs/concepts/storage/persistent-volumes/[persistent volumes] for persistent storage. The size of these volumes is determined by persistence volume claims or PVCs. These PVCs request storage space based on what has been made available to the nodes in your cluster. 
+
+In this document, we will run through the steps to increate the sizes of the persistent volumes used by your internally deployed databases, by increasing the size of our persistent volume claims. This operation should not require any downtime.
```

Comment:
This wasn't true from testing with MediaMath:

`This operation should not require any downtime.`

We had to delete the Postgres/Mongo pods to propagate the change. The kubectl commands showed the right size but `df` was reporting the previous size until the pods were bounced.